app/views/pages/home.html.erb

```diff
@@ -1 +1,55 @@
-<h1>Welcome to Quizzland!</h1>
+<header>
```

Comment:
do you need header only at home page?
if everywhere, then render it in `application.html.erb`